web/packages/teleport/src/services/auth/auth.ts

```diff
@@ -273,12 +269,32 @@ const auth = {
     return api.post(cfg.api.createPrivilegeTokenPath, { secondFactorToken });
   },
 
-  fetchWebauthnChallenge() {
+  async fetchWebauthnChallenge(isMFARequiredRequest?: IsMfaRequiredRequest) {
+    // TODO(Joerger): DELETE IN 16.0.0
+    // the create mfa challenge endpoint below supports
+    // MFARequired requests without the extra roundtrip.
+    if (isMFARequiredRequest) {
+      let isMFARequired: IsMfaRequiredResponse;
+      try {
+        isMFARequired = await checkMfaRequired(isMFARequiredRequest);
```

Comment:
Let's remove `let isMFARequired`, we read it only locally, in `try` clause.
```suggestion
        const isMFARequired = await checkMfaRequired(isMFARequiredRequest);
````